selfdrive/car/docs_definitions.py

```diff
@@ -31,60 +30,84 @@ class Star(Enum):
   EMPTY = "empty"
 
 
-class Harness(Enum):
```

Comment:
I think we still want the harness connectors to have their own enum, perhaps subclass instead

Reply:
subclass should be fine, typing was a problem if it was a separate enum